scripts/install-angular-snapshot.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Script that modifies the project `package.json` by setting the version for all Angular
+# dependencies to the Github snapshot builds. Afterwards, if the `--only-save` flag is not set,
+# the script will also run NPM to install the new versions.
+
+set -e
+
+# Go to the project root directory
+cd $(dirname $0)/../
+
+searchRegex='(@angular\/(.*))":\s+".*"'
+searchReplace='\1": "github:angular\/\2-builds"'
+
+sed -i -r "s/${searchRegex}/${searchReplace}/g" package.json
+
+if [[ ${*} != *--only-save* ]]; then
```

Comment:
@jelbourn Should be ready for review now. 

Also, I wasn't sure whether we should just remove that option completely and don't install at all, but couldn't think of a good name for the script in that case.

Right now the idea is that people can also just run the script locally and it will install the dependencies.